test/karma.conf.js

```diff
@@ -0,0 +1,32 @@
+// This file only hook up on require calls to transpile the TypeScript.
+// If you're looking at this file to see Karma configuration, you should look at
+// karma.config.ts instead.
```

Comment:
After seeing this, my feelings are pretty mixed. It adds this extra layer of indirection and isn't what people expect when looking at a `karma.conf.js`. And the code in the config file is simple enough that you don't get a huge benefit from TS.

@robertmesserle What do you think?